doc/semifp.xml

```diff
@@ -0,0 +1,44 @@
+#############################################################################
+##
+#W  semigrp.xml
+#Y  Copyright (C) 2016                                  James D. Mitchell
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+<#GAPDoc Label="ParseRelations">
+<ManSection>
+  <Oper Name = "ParseRelations" Arg = "gens, rels"/>
+  <Returns>A list of pairs of semigroup elements.</Returns>
+  <Description>
+    <C>ParseRelations</C> can be used to convert a string describing relations
+    for a semigroup to the list of pairs of semigroup elements it represents.
+    e.g. the string  x y^2 = (a^2b)^2x, z=xy = yzx would become
+    [ [ x*y^2, (a^2*b)^2*x ], [ z, x*y ], [ x*y, y*z*x ] ].
+    <P/>
+
+    <A/>gens<A/> must be a list of generators for a free semigroup, each being
```

Comment:
Don't start sentences with symbols.

Reply:
If the parser ignores whitespace, then add that to the documentation.

Reply:
Does `ParseRelations` check if the symbols in the string are valid? What if we use `FreeSemigroup("1", "2");`? Probably this shouldn't be allowed.